tests/wpt/metadata/html/semantics/forms/form-submission-0/form-submission-algorithm.html.ini

```diff
@@ -14,3 +14,6 @@
   [Submission URL should always have a non-null query part]
     expected: FAIL
 
+  [If firing submission events flag of form is true, then return]
+    expected: FAIL
```

Comment:
This is issue with submission algorithm, it didn't manifest before, because `oninvalid` wasn't implemented.